crates/iota-transactional-test-runner/syntax.md

```diff
@@ -71,6 +71,74 @@ in the respective sections.
 
 ### `run-graphql`
 
+Allows to execute GraphQL queries with optional options and returns the output.
+
+#### Syntax:
+
+```
+//# run-graphql [OPTIONS]
+<GraphQL-query>
+```
+
+#### Options:
+
+```
+--show-usage: Displays usage information for the command.
+--show-headers: Includes HTTP headers in the output.
+--show-service-version: Displays the version of the service handling the GraphQL query.
+--cursors <cursor-list>: Specifies a list of cursors to be used within the query.
+```
+
+#### Query Interpolation
+
+The command supports **query interpolation**, allowing you to dynamically replace parts of the GraphQL query at runtime.
+It supports the following placeholders:
+
+1. **Object Placeholders**
+   - **Syntax**: `@{obj_x_y}` or `@{obj_x_y_opt}`
```

Comment:
Could the example also include this syntax?

Reply:
Added an example to better showcase it.